src/main/java/Duke.java

```diff
@@ -1,10 +1,89 @@
+import java.util.Scanner;
 public class Duke {
+    static final int MAX_SIZE = 100;
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printsGreeting();
+        mimicMessage();
+
+    }
+
+    private static void printsGreeting() {
```

Comment:
i think we can use a singular verb here, to print the greetings
```suggestion
    private static void printGreeting() {
```